### PR TITLE
fix: hardcoded dmg path

### PR DESCRIPTION
**Pull request by Swung0x48**
I'm not familiar with rust. I dunno whether this is an ok way to convert path to string?

### CODE REVIEW

**Comment by jkcoxson** [comment]
Could you remove the /dmg/ ? Imo it should just use the folder from the config without creating another folder called dmg.

File: src/client.rs [hunk 1]
```diff
@@ -304,18 +306,20 @@ impl Client {
...
 
-        // Get current directory
-        let current_dir = match std::env::current_dir() {
-            Ok(dir) => dir.canonicalize().unwrap().to_string_lossy().to_string(),
-            Err(_) => {
-                return Err("Unable to get current directory".to_string());
-            }
-        };
+        // // Get current directory
+        // let current_dir = match std::env::current_dir() {
```
Comment (jkcoxson) [inline src/client.rs]: If this isn't needed anymore, could you remove this?

File: src/client.rs [hunk 2]
```diff
@@ -366,7 +361,7 @@ impl Client {
             }
         };
         // Create tmp path
-        let tmp_path = format!("{}/dmg/tmp", &current_dir);
+        let tmp_path = format!("{}/dmg/tmp", &self.dmg_path);
```
Comment (jkcoxson) [inline src/client.rs]: /dmg needs to be removed here

File: src/client.rs [hunk 3]
```diff
@@ -395,11 +390,11 @@ impl Client {
         }
         // Move DMG to JIT Shipper directory
         let ios_dmg = dmg_path.join("DeveloperDiskImage.dmg");
-        std::fs::rename(ios_dmg, format!("{}/dmg/{}.dmg", &current_dir, ios_version)).unwrap();
+        std::fs::rename(ios_dmg, format!("{}/dmg/{}.dmg", &self.dmg_path, ios_version)).unwrap();
```
Comment (jkcoxson) [inline src/client.rs]: /dmg needs to be removed here

File: src/client.rs [hunk 4]
```diff
@@ -395,11 +390,11 @@ impl Client {
...
         let ios_sig = dmg_path.join("DeveloperDiskImage.dmg.signature");
         std::fs::rename(
             ios_sig,
-            format!("{}/dmg/{}.dmg.signature", &current_dir, ios_version),
+            format!("{}/dmg/{}.dmg.signature", &self.dmg_path, ios_version),
```
Comment (jkcoxson) [inline src/client.rs]: here

File: src/client.rs [hunk 5]
```diff
@@ -411,7 +406,7 @@ impl Client {
         );
 
         // Return DMG path
-        Ok(format!("{}/dmg/{}.dmg", &current_dir, ios_version))
+        Ok(format!("{}/dmg/{}.dmg", &self.dmg_path, ios_version))
```
Comment (jkcoxson) [inline src/client.rs]: here

**Review by jkcoxson — COMMENTED** [review]
There are still some /dmg's that will cause it to fail finding.

File: src/client.rs (hunk 1 above)
Comment (Swung0x48) [reply src/client.rs]: alright just fixed that

**Review by Swung0x48 — COMMENTED** [review]
now free of /dmg's, it seems

**Comment by jkcoxson** [comment]
Looks good! This will be pushed to prod next time there's an update.

**Pull request merged**